test/unit/util/save-async-stack.js

```diff
@@ -13,7 +13,11 @@ describe('saveAsyncStack', function () {
     };
     saveAsyncStack(fakeInstance, 1);
 
```

Comment:
Could you please add `test/unit/util/nanoid.js` and `test/unit/util/save-async-stack.js` into  `test/db-less-test-suite.js`?

Reply:
Done